content/docs/components/hyperparameter-tuning/katib-config.md

```diff
@@ -0,0 +1,139 @@
++++
+title = "Katib config overview"
+description = "How to make changes in Katib config"
+weight = 40
++++
+
+This page describes information about [Katib config](https://github.com/kubeflow/katib/blob/master/manifests/v1alpha3/katib-controller/katib-config.yaml).
+
+Katib config is the Kubernetes [Config Map](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/) that contains information about:
+
+1. Current [Metrics Collectors](/docs/components/hyperparameter-tuning/experiment/#metrics-collector) (`key = metrics-collector-sidecar`)
+2. Current [Algorithms](/docs/components/hyperparameter-tuning/experiment/#search-algorithms-in-detail) (Suggestions) (`key = suggestion`).
+
+Katib Config Map must be deployed in [`KATIB_CORE_NAMESPACE`](/docs/components/hyperparameter-tuning/env-variables/#katib-controller) namespace with `katib-config` name. Katib controller parses Katib config when you submit Experiment.
+
+You can edit this Config Map even after deploying Katib.
+
+If you deploy Katib in Kubeflow namespace, to edit Katib config run this:
+
+`kubectl edit configMap katib-config -n kubeflow`
+
+## Metrics Collector Sidecar settings
+
+These settings are related to Katib Metrics Collectors, where:
+
+- key = `metrics-collector-sidecar`
+- value = corresponding settings JSON for each Metrics Collector.
+
+Example for the `File` Metrics Collector with all settings:
+
+```json
+metrics-collector-sidecar: |-
+{
+  "File": {
+    "image": "gcr.io/kubeflow-images-public/katib/v1alpha3/file-metrics-collector",
+    "imagePullPolicy": "Always",
+    "resources": {
+      "requests": {
+        "memory": "200Mi",
+        "cpu": "250m",
+        "ephemeral-storage": "200Mi"
+      },
+      "limits": {
+        "memory": "1Gi",
+        "cpu": "500m",
+        "ephemeral-storage": "2Gi"
+      }
+    }
+  },
+  ...
+}
+```
+
+All of these settings except **`image`** can be omitted. If you don't specify any other settings, default value is be set.
+
+1. `image` - Docker image name for the `File` Metrics Collector.
+
+   **Must be specified**.
+
+2. `imagePullPolicy` - `File` Metrics Collector container [image pull policy](https://kubernetes.io/docs/concepts/configuration/overview/#container-images).
```

Comment:
For flexibility for future updates to the content, use `1.` for all list items, rather than incrementing the list item markup.